### PR TITLE
2 0 workflow voters

### PR DESCRIPTION
**Pull request by ngilain**
[OO-FEATURE] Add workflow voters
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2024
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/699
https://github.com/open-orchestra/open-orchestra-model-interface/pull/264
https://github.com/open-orchestra/open-orchestra/pull/1109
https://github.com/open-orchestra/open-orchestra-front-demo/pull/324

### CODE REVIEW

File: Workflow/Security/Authorization/Voter/AbstractWorkflowVoter.php [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    /**
+     * SuperAdmin can use pass a subject from $fromStatus to $toStatus if such a transition exists
+     *
+     * @param StatusInterface $fromStatus
+     * @param StatusInterface StatusInterface $toStatus
+     *
+     * @return boolean
+     */
+    protected function voteForSuperAdmin(StatusInterface $fromStatus, StatusInterface $toStatus) {
+        if ($this->workflowRepository->hasTransition($fromStatus, $toStatus)) {
```
Comment (alavieille) [inline Workflow/Security/Authorization/Voter/AbstractWorkflowVoter.php]: return $this->workflowRepository->hasTransition($fromStatus, $toStatus)

**Comment by ngilain** [comment]
@alavieille : update done

File: Workflow/Security/Authorization/Voter/NodeWorkflowVoter.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * Class NodeWorkflowVoter
+ *
+ * Voter checking rights on node transitions
+ */
+class NodeWorkflowVoter extends AbstractWorkflowVoter
+{
+    /**
+     * @return array
+     */
+    protected function getSupportedClasses()
```
Comment (alavieille) [inline Workflow/Security/Authorization/Voter/NodeWorkflowVoter.php]: can you rebase because now this method is refacto

**Comment by ngilain** [comment]
@alavieille : rebase done

**Pull request merged**